labs/shipped-import-project/1.md

```diff
@@ -0,0 +1,34 @@
+# Shipped - Import Existing project to shipped
+
+1 Choose Exiting Project
+You can select your git repo you want to integrate with shipped, For this example we have sample project 'goproject' written in golang. 
```

Comment:
We should suggest user to fork one of the buildpack repo (in this case golang buildpack repo ), else below Dockerfile may not work.